.github/workflows/test.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: Check out code.
         uses: actions/checkout@v1
         with:
-          fetch-depth: 1
+          fetch-depth: 10
```

Comment:
could you please teach me about this change.

Reply:
sometimes this workflow fails on master branch because of failing to fetch commits.
https://github.com/vdaas/vald/actions/runs/164399448
To prevent this, it is better to fetch not only the latest commit.